README.md

```diff
@@ -30,6 +32,7 @@ In a Rails application, how is the router related to controller actions?
 
 ```text
 Your answer...
+The url is defined by the router.
```

Comment:
The urls that our application can respond to are defined by the router.  What does the router do in the request/response cycle?  How does it relate to controller actions?